ci-operator/jobs/infra-periodics.yaml

```diff
@@ -269,6 +269,7 @@ periodics:
         sources:
         - secret:
             name: cluster-secrets-aws
+    serviceAccountName: config-updater
```

Comment:
I wonder if we should use a different account (maybe a dedicated one)? I don't think deprovisioning AWS counts as "updating config".

Reply:
/hold

Ack. Let me create a new one.
Does `ipi-deprovisioner` sound like a good `sa` name to you?

Reply:
SGTM

Reply:
/hold cancel

`sa: ipi-deprovisioner` and its `rbac` is added.